recipe/meta.yaml

```diff
@@ -21,13 +21,18 @@ requirements:
     - pip
     - setuptools
     - cppy
```

Comment:
Should be `cppy>=1.1.0 `https://github.com/nucleic/kiwi/blob/7fee18f6978ed88d09cd8fc39053d4ec2b7bae6e/setup.py#L91

Reply:
The pinnings for `cppy` that has been updated

https://github.com/AnacondaRecipes/kiwisolver-feedstock/blob/update_to_1.3.2/recipe/meta.yaml#L24